utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js

```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server'
+import { version } from 'dd-trace/package.json'
+import { libddwaf_version as libddwafVersion } from '@datadog/native-appsec/package.json'
```

Comment:
```suggestion
// eslint-disable-next-line import/no-extraneous-dependencies
import { libddwaf_version as libddwafVersion } from '@datadog/native-appsec/package.json'
```

Reply:
Actually, if this is used and it's not an explicit dependency it would fail (hence the eslint complaint). When we mount dd-trace that will be the case.